lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -118,16 +104,17 @@ describe('SimpleOfflineCache instance', function () {
     var request = new Request('/index.html?param=zzz');
 
     var cacheObj = {
-      match: sinon.stub().returns(Promise.resolve(null)),
-      put: sinon.stub().returns(Promise.resolve())
+      match: sinon.stub().returns(Promise.resolve(null))
     };
-    sinon.stub(self.cacheHelper, 'getCache').returns(
+    // Clean the cached request
+    delete offline.cacheRequest;
```

Comment:
It's very strange you need to delete the request from a new instance of `SimpleOfflineCache`. I would prefer you to assert about `offline.cacheRequest` to be `undefined` just after creating the instance rather than modifing it. Moreover, this is an internal so I would not touch it at all.